x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild

```diff
@@ -26,6 +29,8 @@ DEPEND="
 	virtual/pkgconfig
```

Comment:
Could you move `virtual/pkgconfig` to BDEPEND, as it's EAPI=7? Thanks!

Reply:
Thank you! Sure. Done.
Also changed the revision as Mikle wants :)